src/ansible_navigator/utils.py

```diff
@@ -384,6 +385,22 @@ def pascal_to_snake(obj):
     return obj
 
 
+def path_is_relative_to(child: Path, parent: Path) -> bool:
+    """Return True if the path is relative to another path or False.
+
+    :param child: The path that may be a child
+    :param parent: The path that may be a parent
+    :returns: Indicates the child is a child of the parent
+    """
+    if sys.version_info >= (3, 9):
+        return child.is_relative_to(parent)
+    try:
+        child.relative_to(parent)
+        return True
+    except ValueError:
```

Comment:
What a weird API (on pathlib's part).

Reply:
for sure.........  I had to name it child and parent just to make sense of it, and add the unit test so I can remember what the heck they were doing.........  I always considered my parents relatives, and I think they probably consider me one.

Reply:
Well that, but the whole... using exceptions as a way to say "nope, not relative"